ReactWindows/ReactNative.Net46/Tracing/Tracer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace ReactNative.Tracing
+{
+    class NullDisposable : IDisposable
+    {
+        public void Dispose()
+        {
+
+        }
+    }
+
+    public class NullLoggingActivityBuilder
```

Comment:
>NullLoggingActivityBuilder [](start = 17, length = 26)

Can you ensure you check the documentation box in the project settings for ReactNative.Net46 for all configurations and platforms?  We want to get compile time warnings for missing documentation on public surfaces like this.